Fedoraware/Fedoraware-TF2/src/Hooks/Detours/BaseClientDLL_DispatchUserMessage.cpp

```diff
@@ -121,6 +114,10 @@ MAKE_HOOK(BaseClientDLL_DispatchUserMessage, Utils::GetVFuncPtr(I::BaseClientDLL
 
 		case TextMsg:
 		{
+			if (F::NoSpread.ParsePlayerPerf(msgData)) {
```

Comment:
Doesn't this prevent AntiAutobalance from running, if NoSpread is enabled?

Reply:
Hmm, i haven't tested that but it should definitely work if the nospread is synced. It might not work if its syncing but i guess you can just move it so it runs after anti-autobalance